greedy_approach/Knapsack.C

```diff
@@ -0,0 +1,62 @@
+
+#include <bits/stdc++.h> 
+
+using namespace std; 

```

Comment:
A C program cannot use the `std` namespace (it's only for C++).
The appropriate library should be the `stdio.h`, and using `printf` and `scanf`.